chef/cookbooks/updater/recipes/default.rb

```diff
@@ -40,6 +40,10 @@
       zypper_command += " --auto-agree-with-licenses"
     end
 
+    execute "refresh PTF repository" do
+      command "zypper --non-interactive --gpg-auto-import-keys refresh -fr PTF"
+    end
```

Comment:
This should have a `only_if` guard to protect in case there is no PTF repo on the node.